Client/Frontend/Browser/Browser.swift

```diff
@@ -53,6 +53,10 @@ class Browser: NSObject {
     /// The last title shown by this tab. Used by the tab tray to show titles for zombie tabs.
     var lastTitle: String?
 
+    /// The user agent override used for the last request, reload or navigation. Note that this property needs to
+    /// be managed by the web view's navigation delegate.
+    var lastCustomUserAgent: String? = nil
```

Comment:
How about just having this be a boolean, `desktopSite`? It doesn't look like the string is actually used anywhere.